packages/e2e-test-utils/src/visitAdminPage.js

```diff
@@ -35,7 +35,10 @@ import {
  * @param {string} [query] String to be serialized as query portion of URL.
  */
 async function visitAdminPage(adminPath, query = '') {
-  await page.goto(createURL(join('wp-admin', adminPath), query));
+  await page.goto(createURL(join('wp-admin', adminPath), query), {
+    waitUntil: 'load',
```

Comment:
That's the default, FWIW. Try other options to see if it makes a difference.

https://pptr.dev/api/puppeteer.page.goto